test/acceptance/graphql_resolvers/local_aggregate_matrix_groupby_test.go

```diff
@@ -899,10 +881,969 @@ func aggregateArrayClassWithGroupByTest(t *testing.T) {
 				extracted := extractArrayClassGroupByResult(result)
 				expected := tc.expected.([]interface{})
 
-				assert.Len(t, extracted, len(expected))
-				for _, singleExpected := range expected {
-					assert.Contains(t, extracted, singleExpected, "expected group not found in results array")
-				}
+				assert.ElementsMatch(t, expected, extracted)
+			})
+		}
+	})
+}
+
+func aggregateCityClassWithGroupByTest(t *testing.T) {
+	t.Run("aggregate City with group by city area", func(t *testing.T) {
+		h := &gqlAggregateResponseHelper{}
+		testCasesGen := &aggregateCityTestCases{}
+
+		allResults := []interface{}{
```

Comment:
Could you give a little context what exactly you're testing? Just with the numbers it's difficult to understand what exactly is tested

Reply:
i've changed the way assertions are being performed. basically whole response is verified - each aggregated value is compared with expected. please let me know if it requires more explanation.

Reply:
it's clearer now, thx!